apps/web/src/auth/auth.const.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: Add a novu prefix to the local storage key
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ Is this something that will be addressed in this PR or separately?

Reply:
I moved this comment along with the constant, so would likely aim for a future PR so that the original commenter can do so or explain why this hasn't been done already. I figure it's a matter of needing to modify the token retrieval method to be backwards compatible for the max duration that the `auth_token` could be alive, so it may be:
a change --> wait 1 month --> remove the old `auth_token` retrieval